orttraining/orttraining/core/graph/training_op_defs.cc

```diff
@@ -2494,6 +2494,24 @@ Example 4:
            {{"X_1"}, "Cos", {"X"}},
            {{"dX"}, "Mul", {"X_1", "dY"}}}));
 
+  ONNX_CONTRIB_OPERATOR_SCHEMA(CosGrad)
+    .SetDomain(kOnnxDomain)
+    .SinceVersion(9)
+    .SetSupportLevel(OpSchema::SupportType::EXPERIMENTAL)
```

Comment:
Couple of questions:
1. why are we registering this to ONNX domain and not contrib domain?
2. Why version 9? Cos op latest version is 7
3. Why experimental? experimental support level has been deprecated from ONNX several years back.

@prathik I think you simply followed the trend in this file. @pengwa , @iK1D : Do you have the historical context for this? Can you help answer my questions?

Reply:
@askhade For #3,  looks like all the ops have SetSupportLevel(OpSchema::SupportType::EXPERIMENTAL)